src/AreaChart/AreaSeries/Line.tsx

```diff
@@ -148,17 +148,12 @@ export const Line: FC<Partial<LineProps>> = ({
   const enter = useMemo(() => {
     const linePath = getLinePath(coords);
 
-    let strokeDasharray = '';
```

Comment:
Can you test if the line chart works when you transition it? AKA default value then change the value. I think thats what this code is for.

Reply:
I've checked all LineChart stories with these changes and without - and I don't see any difference

Reply:
Ya - Its not animating - it just jumps to the next point. Watch the video here, the first one animates to the next point ( master ) the last one jumps to it ( your branch ).

https://app.screencast.com/hMr9yN5vGWIem